docs/Tutorials/Privacy/Configuring-Multi-Tenancy.md

```diff
@@ -43,48 +46,100 @@ in `.pem` format, belongs to the operator who uses the key pair to authenticate
     This step is not required when using [username and password authentication] to create the
     required JWTs.
 
-## 2. Generate Orion keys
-
-In the `Node-1/Orion` directory,
-[generate a public/private key pair for each tenant](Configuring-Privacy.md#3-generate-orion-keys).
-
-Name the key pair `nodeKey2` and `nodeKey3`.
-
-## 3. Update the password file
+## 2. Generate Tessera keys
 
-Update `passwordFile` in the `Node-1/Orion` directory by adding each password used to generate the
-Orion keys on a new line.
+In the `Node-1/Tessera` directory,
+[generate a public/private key pair for each tenant](Configuring-Privacy.md#2-generate-tessera-keys).
 
-You require separate passwords for each key pair, even if the passwords are identical.
+!!! note
 
-## 4. Update the Orion configuration file
+    The instructions creates an unlocked private key, meaning you do not need a password to decrypt
+    the private key file.
 
-In the `Node-1/Orion` directory, update the `orion.conf` file by adding the new key pairs:
+Name the key pair `nodeKey2` and `nodeKey3`.
 
-```bash
-nodeurl = "http://127.0.0.1:8080/"
-nodeport = 8080
-clienturl = "http://127.0.0.1:8888/"
-clientport = 8888
-publickeys = ["nodeKey.pub", "nodeKey2.pub", "nodeKey3.pub"]
-privatekeys = ["nodeKey.key", "nodeKey2.key", "nodeKey3.key"]
-passwords = "passwordFile"
-tls = "off"
+## 3. Update the Tessera configuration file
+
+In the `Node-1/Tessera` directory, update the `tessera.conf` file by adding the new key pairs:
+
+```json
+{
+   "mode": "orion",
+   "useWhiteList": false,
+   "jdbc": {
+       "username": "sa",
+       "password": "",
+       "url": "jdbc:h2:./target/h2/tessera1",
+       "autoCreateTables": true
+   },
+   "serverConfigs":[
+       {
+           "app":"ThirdParty",
+           "enabled": true,
```

Comment:
enabled is ignored so I would remove from here

Reply:
Fixed